docker/algod/source/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.12
+FROM golang:1.16.11
```

Comment:
Upgrade to latest version. Was failing without due to stale SSL certs.

Reply:
👍